src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/ClassDefHandler.java

```diff
@@ -93,6 +97,24 @@ public void checkIndentation() {
         super.checkIndentation();
     }
 
+    /**
+     * Get the right parenthesis portion of the expression we are handling.
+     *
+     * @return the right parenthesis expression
+     */
+    private DetailAST getRecordComponentsRightParen() {
+        return getMainAst().findFirstToken(TokenTypes.RPAREN);
```

Comment:
Isn't this just duplicating `getRightParen`?

Reply:
See above.